### PR TITLE
LinkedIn Activity PD

### PR DESCRIPTION
**Pull request by zoefarrell**

### CODE REVIEW

File: module5/lessons/Week5/LinkedInActivityPD.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+2. Go to your Linkedin profile and click the blue box with a “C” in it on the right hand side of the page.
+    * It will prompt you to make an account to view suggested improvements.
+3. Make the recommended changes to your profile to improve it - you have until lunch on Thursday to make changes
+4. Add up the percentages of each team member and average them.
+
+The team spokesperson will DM your instructors with your group’s average score and any bonus points earned (Each bonus point counts as adding a percentage point to your team average!). Average score is due at noon this Thursday!
+
+The team with the highest average overall score will be the winners!
+
+### Bonus Points!
```
Comment (zoefarrell) [inline module5/lessons/Week5/LinkedInActivityPD.md:23]: I tried out the CareerFlow thing and it's pretty cool! This is what the output looks like.
Screenshot 2023-10-19 100922

Any other bonus things you can think of that we'd want them engaging with?

Comment (zoefarrell) [reply module5/lessons/Week5/LinkedInActivityPD.md:23]: Screenshot 2023-10-19 101016

File: module5/lessons/Week5/LinkedInActivityPD.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+---
+layout: page
+title: LinkedIn Activity
+---
+
+Today is all about returning to your LinkedIn. You will be polishing up your profile and engaging with the broader developer community to make sure your LinkedIn best represent you!
```
Comment (zoefarrell) [inline module5/lessons/Week5/LinkedInActivityPD.md:6]: PR feedback!

File: module5/lessons/Week5/LinkedInActivityPD.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+## Careerflow Competition! 
+
+Instructions for all team members:
+
+1. Install the [Careerflow Chrome Extension](https://www.google.com/url?q=https://chrome.google.com/webstore/detail/careerflow-ai-linkedin-op/iadokddofjgcgjpjlfhngclhpmaelnli&sa=D&source=calendar&ust=1695826733869948&usg=AOvVaw03e0L95JHMmx0VCrDyAm3g)
+2. Go to your Linkedin profile and click the blue box with a “C” in it on the right hand side of the page.
+    * It will prompt you to make an account to view suggested improvements.
+3. Make the recommended changes to your profile to improve it - you have until lunch on Thursday to make changes
+4. Add up the percentages of each team member and average them.
```
Comment (memcmahon) [inline module5/lessons/Week5/LinkedInActivityPD.md:16]: I wonder if we need to include an example here?  I don't think the math is difficult, but just the words 'percentage' and 'average' in the same sentence might be daunting.

**Review by memcmahon — APPROVED** [review]
This is so great!!! I had never heard of the Career Flow extension (I'm honestly scared to activate it on my profile 😆 )

I love the gamification of this activity!

**Comment by zoefarrell** [comment]
> This is so great!!! I had never heard of the Career Flow extension (I'm honestly scared to activate it on my profile 😆 )
> 
> I love the gamification of this activity!

This is one of the cool things I pulled from the work Tracy has been doing! The students will definitely have higher scores than me at the end of this 😆

File: module5/lessons/Week5/LinkedInActivityPD.md (hunk 3 above)
Comment (zoefarrell) [reply module5/lessons/Week5/LinkedInActivityPD.md:16]: I like that idea of showing a quick example! Will add :)

**Pull request merged**